<launch>
  <node name="nmpc" pkg="fw_ctrl" type="fw_nmpc"/>
  <param name="/nmpc/loop_rate" type="double" value="10.0" />
  <param name="/nmpc/time_step" type="double" value="0.1" />

  <arg name="fake_signals" default="0" />
  <param name="/nmpc/fake_signals" type="int" value="$(arg fake_signals)"/>
</launch>
